src/types/options.d.ts

```diff
@@ -11,7 +11,7 @@ export type Props = 'info' | 'sitelinks' | 'sitelinks/urls' | 'aliases' | 'label
 export type UrlResultFormat = 'xml' | 'json';
 export type WmLanguageCode = typeof languages[number]
 
-export type ApiQueryParameters = Record<string, string | number>
+export type ApiQueryParameters = Record<string, string | number | boolean>
```

Comment:
As the docs state that any value will be treated as true for boolean (and the value not being there as false) the value false will result in a true which is not what the user wanted. So only true or omitting it is possible

```suggestion
export type ApiQueryParameters = Record<string, string | number | true>
```